### PR TITLE
Add support for lists of Values in compute

### PR DESCRIPTION
**Pull request by jcrist**
```python
>>> a = value(1)
>>> b = a + 1
>>> c = a + 2
>>> compute(a, [b, c])
(1, [2, 3])
```

### CODE REVIEW

File: dask/imperative.py [hunk 1]
```diff
@@ -193,7 +193,10 @@ def compute(*args, **kwargs):
     >>> c = a + 3
     >>> compute(b, c)  # Compute both simultaneously
     (3, 4)
+    >>> compute(a, [b, c])
+    (1, [3, 4])
     """
+    args = [value(a) for a in args]
```
Comment (mrocklin) [inline dask/imperative.py]: We should probably make `value` idempotent

File: dask/imperative.py [hunk 2]
```diff
@@ -345,6 +348,8 @@ def value(val, name=None):
     AttributeError("'list' object has no attribute 'not_a_real_method'")
     """
 
+    if isinstance(val, Value):
+        return val
```
Comment (mrocklin) [inline dask/imperative.py]: Oh, right.

**Comment by mrocklin** [comment]
+1

File: dask/imperative.py (hunk 2 above)
Comment (jcrist) [inline dask/imperative.py]: :)

**Pull request merged**